src/styles/card.css

```diff
@@ -0,0 +1,117 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: flex-start;
+
+  font-family: Roboto, sans-serif;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  padding-top: 32px;
+}
+
+.card__text {
+  padding: 40px 16px 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.card__code {
+  /* padding-top: 4px; */
+  padding-left: 16px;
+  flex-basis: 100%;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__stars {
+  display: flex;
+  flex-basis: 100%;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 16px;
+  padding-left: 16px;
+}
+
+.card__review {
+  color: #060b35;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 16px;
+
+  display: flex;
+  align-items: flex-end;
+  text-align: right;
+  padding-right: 16px;
+}
+
+.card__price {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  padding: 24px 16px 0;
+
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+
+  display: flex;
+  align-items: center;
```

Comment:
I'm not sure, but it'll be better to set align-items to card__price (it'll center both elements, even if there's no need to center card__price-sum)